src/OpenTelemetry/Trace/SamplingResult.cs

```diff
@@ -42,6 +43,7 @@ public SamplingResult(bool isSampled)
         {
```

Comment:
Do we not need an overload for this one?

Reply:
its optional. if there is a need, we can always add more overloads in future.